File: python/guessing_game/guessing_game.py
import random

print ("Guess the number!!! \n")

secret_number = random.randint(1,100)

while True:
    guess = int(input("Please input your guess \n"))
    print("Your Guess: ", guess)
    if (guess > secret_number):
        print("Too Big!")
    elif (guess < secret_number):
        print("Too Small!")
    else:
        print("You Win!")
        break        

File: Go/guessingGame/guessingGame.go
package main

import (
	"fmt"
	"math/rand"
)

func main() {
	fmt.Println("Guess the Number!!! ")
	secretNumber := rand.Intn(100)
	guess := 0
	for {
		fmt.Println("Please input your guess ")
		_, err := fmt.Scan(&guess)
		if err != nil {
			fmt.Println(err)
		}
		fmt.Println("Your guess is", guess)

		if guess > secretNumber {
			fmt.Println("Too Big!")
		}
		if guess < secretNumber {
			fmt.Println("Too Small!")
		}
		if guess == secretNumber {
			fmt.Println("You Win")
			break
		}

	}

}
